lms/services/upsert.py

```diff
@@ -2,16 +2,18 @@
 
 from typing import List
 
+from sqlalchemy import column, tuple_
 from sqlalchemy.dialects.postgresql import insert
 from zope.sqlalchemy import mark_changed
 
 
-def bulk_upsert(
+def bulk_upsert(  # pylint:disable=too-many-arguments,
```

Comment:
Instead of a parameter here we could a second `bulk_upsert_returning` function that uses the existing one.